src/opf/sdpwrm.jl

```diff
@@ -42,9 +42,9 @@ function build_opf(::Type{SDPOPF}, data::OPFData, optimizer;
     =#
 
     # voltage magnitude and product
-    WR_start = zeros(N, N) + I
+    WR_start = zeros(T, N, N) + I
     @variable(model, WR[i=1:N, j=1:N], Symmetric, start=WR_start[i,j])
-    @variable(model, WI[1:N, 1:N] in SkewSymmetricMatrixSpace(), start=0.0)
+    @variable(model, WI[1:N, 1:N] in SkewSymmetricMatrixSpace(), start=T(0.0))
```

Comment:
AFAIK, start values will not changed anything if we use an interior-point solver like Clarable or Mosek.
Note that it also requires `2*N^2` memory (because JuMP will save those start values)

Reply:
I'm in favor of removing them